repo/packages/C/crate/0/config-sample.json

```diff
@@ -0,0 +1,10 @@
+{
+  "crate/version": "0.48.1",
+  "crate/cluster-name": "dcos",
+  "framework/zookeeper": "<mesos-master>:2181",
+  "crate/data-path": "/home/crate/data",
+  "crate/resource/cpus": 1.0,
+  "crate/resource/memory": 2048,
+  "crate/resource/heap": 1024
+}
```

Comment:
what is the purpose of this file?  can you remove it?  It seems like something for a gist, but not for this repo.